### PR TITLE
Gang

### PR DESCRIPTION
**Pull request by g4s8**
zerocracy/farm#366 - person fields for gang

### CODE REVIEW

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#266](https://github.com/zerocracy/datum/issues/266) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#266](https://github.com/zerocracy/datum/issues/266) is assigned to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx). The budget is 15 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§27](http://datum.zerocracy.com/pages/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

**Comment by 0crat** [comment]
@fabriciofx[/z](https://www.0crat.com/u/fabriciofx) this job was assigned to you 5 days ago. It will be taken away from you soon, unless you close it, see [§8](http://datum.zerocracy.com/pages/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by 0crat** [comment]
@fabriciofx[/z](https://www.0crat.com/u/fabriciofx) this job was assigned to you 8 days ago. It will be taken away from you soon, unless you close it, see [§8](http://datum.zerocracy.com/pages/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by 0crat** [comment]
The user @fabriciofx[/z](https://www.0crat.com/u/fabriciofx) resigned from [#266](https://github.com/zerocracy/datum/issues/266), please stop working

**Comment by 0crat** [comment]
This pull request [#266](https://github.com/zerocracy/datum/issues/266) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/432023674). The budget is 15 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§27](http://datum.zerocracy.com/pages/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

**Comment by carlosmiranda** [comment]
@yegor256 looks good to merge

File: xsd/pmo/people.xsd [hunk 1]
```diff
@@ -53,6 +53,9 @@ SOFTWARE.
           </xs:restriction>
         </xs:simpleType>
       </xs:element>
+      <xs:element name="reputation" type="xs:integer" minOccurs="0"/>
```
Comment (yegor256) [inline xsd/pmo/people.xsd]: @g4s8 why `minOccurs=0`? I think we have to have `minOccurs="1" maxOccurs="1"` and then create an "upgrade" that will set it all to zero for all users.

**Comment by yegor256** [comment]
@g4s8 see above

**Comment by 0crat** [comment]
The user @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) resigned from [#266](https://github.com/zerocracy/datum/issues/266), please stop working

**Comment by g4s8** [comment]
@yegor256 fixed. Please, look again

**Comment by yegor256** [comment]
@g4s8 why not make `jobs` and `projects` mandatory too? In general, the stricter the schema, the better. Flexibility makes code more complex.

**Comment by g4s8** [comment]
@yegor256 done, please see new commit

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:zerocracy/datum#266` is not assigned, can't get start time

**Comment by 0crat** [comment]
The job [#266](https://github.com/zerocracy/datum/issues/266) is now out of scope

**Pull request merged**